EasyPost.Tests/BatchTest.cs

```diff
@@ -114,18 +114,5 @@ public class BatchTest {
 
             batch.GenerateScanForm();
         }
-
-        [TestMethod]
```

Comment:
This PR was not rebased properly. If it was, this section of code would not be included (commits or code). Please rebase properly before we review again.

Reply:
Corrected.